src/main/java/uk/gov/justice/digital/hmpps/community/services/CommunityApiTokenService.java

```diff
@@ -43,26 +42,21 @@ void checkOrRenew() {
     }
 
     private synchronized void renewCachedToken() {
-
-        try {
-            final var exchange = restTemplateResource.exchange("/logon", HttpMethod.POST, deliusLogonEntity, String.class);
-            jwtToken = exchange.getBody();
-            log.info("* * * Renewed token is {}", jwtToken);
-        } catch (final Exception e) {
-            log.error("* * * Exception renewing Delius API token {} ", e.getMessage());
```

Comment:
this should bubble up to the caller rather than be caught and then the delius call fails with a 403 - which is then passed back to client